### ISSUE TITLE
3.14 : segfault in batch conversion: Resize

### ISSUE THREAD

**Issue by fenugrec** [nomacs/nomacs#504]
**Describe the bug**
Selecting "Resize" in the list of adjustments in a batch conversion causes a segfault

**To Reproduce**
1. Tools->Batch Processing
2. Adjustments : click on "Resize"
3. instant segfault

**Expected behavior**
no segfault 

**Screenshots**
I've had problems recompiling nomacs on Arch linux so I'm stuck with the stripped build. backtrace:
```
#0  0x00007ffff74ae1ce in QWidget::show() () at /usr/lib/libQt5Widgets.so.5
#1  0x00007ffff7d01b14 in nmc::DkBatchManipulatorWidget::selectManipulator(QSharedPointer<nmc::DkBaseManipulator>) () at /usr/lib/libnomacsCore.so.3
#2  0x00007ffff7d02096 in nmc::DkBatchManipulatorWidget::selectionChanged(QItemSelection const&) () at /usr/lib/libnomacsCore.so.3
#3  0x00007ffff7cc8d0c in  () at /usr/lib/libnomacsCore.so.3
#4  0x00007ffff7035920 in  () at /usr/lib/libQt5Core.so.5
#5  0x00007ffff6fa83c1 in QItemSelectionModel::selectionChanged(QItemSelection const&, QItemSelection const&) () at /usr/lib/libQt5Core.so.5
#6  0x00007ffff6fb11e1 in QItemSelectionModel::select(QItemSelection const&, QFlags<QItemSelectionModel::SelectionFlag>) () at /usr/lib/libQt5Core.so.5
#7  0x00007ffff7716efc in QListView::setSelection(QRect const&, QFlags<QItemSelectionModel::SelectionFlag>) () at /usr/lib/libQt5Widgets.so.5
#8  0x00007ffff76d7425 in QAbstractItemView::mousePressEvent(QMouseEvent*) ()
    at /usr/lib/libQt5Widgets.so.5
#9  0x00007ffff74b1c7e in QWidget::event(QEvent*) ()
    at /usr/lib/libQt5Widgets.so.5
#10 0x00007ffff755cf9f in QFrame::event(QEvent*) ()
    at /usr/lib/libQt5Widgets.so.5
#11 0x00007ffff6ffe3e3 in QCoreApplicationPrivate::sendThroughObjectEventFilters(QObject*, QEvent*) () at /usr/lib/libQt5Core.so.5
```

**Desktop (please complete the following information):**
 - OS: Arch linux
- nomacs 3.14
- Qt 5.14.2

**Comment by fenugrec** [comment]
Ok, finally able to recompile from git. Cannot reproduce issue anymore.

**Comment by diemmarkus** [comment]
that was fixed, see: #453
